### ISSUE TITLE
Lore & Color Problem

### ISSUE THREAD

**Issue by Julienraptor01** [Cosmetica-cc/Cosmetica#76]
**Describe the bug**
Changing the lore on the web panel reset the color to default
**To Reproduce**
1. Go to https://cosmetica.cc/lore
2. Click on Discord Tag by example
3. Auth
4. See blue color be replaced by grey 😥

**Expected behavior**
Color not to be reset on Lore change

**Screenshots**
*Empty*

**Setup (please complete the following information):**
 - OS : Not Useful
 - Minecraft Version : Not Useful
 - Version : Not Useful

**Additional context**
*Still Empty*

**Comment by Julienraptor01** [comment]
Additional bug, when using ttv as lore it seems to also go back to white after a while

**Comment by Julienraptor01** [comment]
> Additional bug, when using ttv as lore it seems to also go back to white after a while

It happened again
![Screenshot_20221130-002603_Chrome.png](https://user-images.githubusercontent.com/49841713/204670742-eb1e9a86-e114-4883-bfaf-2860a4468c2a.png)
I switched back to my blue and i'll see tommorow if it's back to default
![Screenshot_20221130-002614_Chrome.png](https://user-images.githubusercontent.com/49841713/204670843-6d923f50-1182-4486-b0e8-a08b73c8fbc5.png)

**Comment by Julienraptor01** [comment]
It's a new morning and i lost my color again

**Comment by eyezahhhh** [comment]
Wait it changes the colour back to white at some random time?

**Comment by Julienraptor01** [comment]
> Wait it changes the colour back to white at some random time?

Yes, for some reason but it's just an issue on the website actually because i've constantly kept my blue color IG

It's happening on my phone like nearly every time with this setup
Samsung Galaxy A52s 5G
Android 13 OneUI 5.0
Chrome 108.0.5359.79
